apps/vc-api/src/dtos/conflict-error-response.dto.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021, 2022 Energy Web Foundation
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ConflictErrorResponseDto {
+  @ApiProperty({ example: 409 })
+  statusCode: 409;
+
+  @ApiProperty()
+  message: string;
+
+  @ApiProperty({ example: 'Not Found' })
```

Comment:
Should this be "Conflict"?
```suggestion
  @ApiProperty({ example: 'Conflict' })
```

Reply:
Good finding. Thanks!